src/hotspot/share/cds/cdsConfig.cpp

```diff
@@ -278,6 +278,8 @@ static const char* find_any_unsupported_module_option() {
     "--patch-module"
   };
 
+  assert(false, "GHA TEST FAILURE POINT. REMOVE BEFORE FLIGHT"); // Or wait for Friday and integrate for extra fun.
```

Comment:
The generated summaries look fine. Don't forget to remove this. :)

Reply:
Yanked, we now have a PR ready for integration.